test/parallel/test-dotenv-edge-cases.js

```diff
@@ -200,4 +200,26 @@ describe('.env supports edge cases', () => {
     assert.strictEqual(child.code, 9);
     assert.match(child.stderr, /bad option: --env-file-ABCD/);
   });
+
+  it('should handle invalid syntax in .env file', async () => {
+    const invalidEnvFilePath = fixtures.path('dotenv/invalid-syntax.env');
+    const code = `
+      const { parseEnv } = require('node:util');
```

Comment:
nit: I find it a bit weird to rely on `assert` (and `fs`) to be globally available but not `util`

Reply:
I'll use `process.loadEnvFile` like in upper test cases to keep it simple